src/StdChains.sol

```diff
@@ -20,71 +22,126 @@ abstract contract StdChains {
         string rpcUrl;
     }
 
-    bool private initialized;
+    // chain alias -> chain data
+    mapping(string => Chain) internal _chains;
+    mapping(string => string) internal _defaultRpcUrls;
+    mapping(uint256 => string) internal _idToAlias;
```

Comment:
Let's add the comments back to keep these a bit more self-documenting for future contributors
```suggestion
    // Maps from a chain ID to that chain's alias
    mapping(uint256 => string) internal _idToAlias;
```

Reply:
Do we need a more discouraging naming, so that users don’t shout themselves in the foot? E.g. `_id_to_alias_`.

This is only for devs.

Reply:
What about just making it `private` instead of a weird naming style?

Reply:
https://github.com/foundry-rs/forge-std/pull/247#discussion_r1043412377 - def not the best solution - if @adhusson doesn’t intend to extend the features immediately, a better solution is to keep everything private, as was, and if someone requests it in the future, make changes then. What do you think?

Reply:
Agreed, let's keep as much of the internal implementation `private` as possible to keep the UX/interface as simple/safe as possible, and we can iterate in the future when the need arises